data/src/main/java/com/terning/data/repositoryimpl/SearchRepositoryImpl.kt

```diff
@@ -0,0 +1,24 @@
+package com.terning.data.repositoryimpl
+
+import InternshipAnnouncement
+import com.terning.data.datasource.SearchScrapsDataSource
+import com.terning.data.datasource.SearchViewsDataSource
+import com.terning.domain.repository.SearchRepository
+import javax.inject.Inject
+
+class SearchRepositoryImpl @Inject constructor(
+    private val searchViewsDataSource: SearchViewsDataSource,
+    private val searchScrapsDataSource: SearchScrapsDataSource,
+) : SearchRepository {
+    override suspend fun getSearchViewsList(): Result<List<InternshipAnnouncement>> {
+        return runCatching {
+            searchViewsDataSource.getSearchViews().result.toSearchViewsEntity()
+        }
+    }
```

Comment:
```suggestion
) : SearchRepository {
    override suspend fun getSearchViewsList(): Result<List<InternshipAnnouncement>> = runCatching {
            searchViewsDataSource.getSearchViews().result.toSearchViewsEntity()
        }
    }
```
여기두 이렇게 해주면 좋을 것 같네요!!